### DISCUSSION TITLE
Forgot password/Account unlock on Clients page

### DISCUSSION THREAD

**Discussion by krushnadattaswain** [drakkan/sftpgo#526] [Ideas]
It would be great if we can have Forgot password/Account unlock option in SFTPGo client UI
https://<IP>:<PORT>web/client/login

I can create a PR for that.

**Comment by drakkan** [comment]
Hi,

generally a temporary password is sent via email/sms from password recovery pages, so we should add an email field to the user data structure and make it mandatory to add this feature (and also add a global SMTP configuration). Do you have other ideas? We can also have users with no passwords, for example SFTP users that authenticate only with public keys. These users cannot login using the web UI, we should not generate a temporary password in this case. 

P.S. I'm working on a big refactor to add 2FA via authenticator apps. If you want to avoid merge conflicts please wait for some days before submitting a PR
